.gitignore

```diff
@@ -7,4 +7,4 @@
 # local build of Javadoc (cf. docs/javadoc/README.md)
 /javadoc
 # local build of docs with mkdocs
-/site
+/site
```

Comment:
All (text) files in Kitodo.Production should end with an empty new line. Even GitHub is showing an red symbol on the change diff if this is not the case.

Reply:
Thx. I changed this line so nothing changed. :)